src/main/java/duke/commands/AddCommand.java

```diff
@@ -0,0 +1,28 @@
+package duke.commands;
+
+import duke.DukeException;
+import duke.storage.Storage;
+import duke.tasks.Task;
+import duke.tasks.TaskList;
+import duke.ui.Ui;
+
+/**
+ * Adds a task to the task list.
+ */
+public class AddCommand extends Command {
+    private Task toAdd;
+
+    public AddCommand(Task toAdd) {
+        this.toAdd = toAdd;
+    }
+
+    @Override
+    public void execute(TaskList tasks, Ui ui, Storage storage) throws DukeException {
+        System.out.println("~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~");
+        System.out.println("Alright! I added this task: \n" + toAdd.toString() + "\n");
+        tasks.add(toAdd);
```

Comment:
It may be better to swap these 2 lines as the message implies that the task has been successfully added (in case  something goes wrong in `tasks.add(toAdd)`)